packages/devtools/lib/src/timeline/timeline.css

```diff
@@ -169,3 +169,18 @@
     /* 100% height minus the height of the event-details-heading */
     height: calc(100% - 27px);
 }
+
+.checkbox-container {
```

Comment:
This styling seems too generic to be in timeline.css
I would move it to the main css file or use class names that are more timeline specific.